### PR TITLE
Fix/DOC-57: typos and minor improvements

### PR DESCRIPTION
**Pull request by uF4No**
Fixes typos found by Anthony.
Changes all references of "Hello world" to Quickstart tutorial

### CODE REVIEW

File: docs/dev/developer-guides/hello-world.md [hunk 1]
```diff
@@ -251,7 +249,7 @@ async changeGreeting() {
 },
 ```
 
-On the top of the `<script>` tag, you may see the parts that should be filled with the address of the deployed `Greeter` contract and the path to its ABI. Let's fill these fields in the following sections.
+On the top of the `<script>` tag, you may see the parts that should be filled with the address of the deployed `Greeter` contract and the path to its ABI. We will fill these fields in the following sections.
```
Comment (krofax) [inline docs/dev/developer-guides/hello-world.md:252]: Can you change to:

At the top of the `script` tag,  you may see the parts that should be filled with the address of the deployed `Greeter` contract and the path to its `ABI`. We will fill these fields in the following sections.

File: docs/dev/fundamentals/testnet.md [hunk 2]
```diff
@@ -29,9 +29,9 @@ All the existing SDKs for Ethereum will work out of the box and your users will
 
 The only other place where using zkSync SDK is required is during contract deployment. This can be easily done through our hardhat plugin.
 
-## Hello World on zkSync
+## Quickstart tutorial on zkSync
```
Comment (krofax) [inline docs/dev/fundamentals/testnet.md:32]: It's a tutorial, we don't have to explicitly add it to the heading, hence "Quickstart on zkSync", sounds like it.

File: docs/dev/fundamentals/testnet.md [hunk 3]
```diff
@@ -29,9 +29,9 @@ All the existing SDKs for Ethereum will work out of the box and your users will
...
 
-Check out our step-by-step [tutorial](../developer-guides/hello-world.md), where you will learn:
+Check out our step-by-step [quickstart tutorial](../developer-guides/hello-world.md), where you will learn:
```
Comment (krofax) [inline docs/dev/fundamentals/testnet.md:34]: Same reason as the one above.
We can say "Check out our [quickstart guide](../developer-guides/hello-world.md), where you will learn:"

File: docs/dev/tutorials/README.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ Learn how to build dApps on the zkSync protocol with our how-to guides
 
 ## Table of contents
 
-- [Hello world](../developer-guides/hello-world.md)
+- [Quickstart tutorial](../developer-guides/hello-world.md)
```
Comment (krofax) [inline docs/dev/tutorials/README.md:7]: And here, just "Quickstart", should be fine.

Comment (uF4No) [reply docs/dev/tutorials/README.md]: fixed

File: docs/dev/fundamentals/testnet.md (hunk 3 above)
Comment (uF4No) [reply docs/dev/fundamentals/testnet.md]: fixed

File: docs/dev/developer-guides/hello-world.md [hunk 5]
```diff
@@ -168,6 +162,10 @@ yarn hardhat deploy-zksync
 
 In the output, you should see the address the contract was deployed to.
 
+Congratulations! You have deployed a smart contract to zkSync! Now you can visit the [zkSync block explorer](https://explorer.zksync.io/) and search you contract address to confirm it was successfully deployed.
+
+You can find all the details about how to verify your smart contract using zkSync block explorer [here](../../api/tools/block-explorer/contract-verification.md).
```
[the remark this answers is not in the document]
Comment (uF4No) [reply docs/dev/developer-guides/hello-world.md]: fixed

**Review by krofax — APPROVED** [review]

**Pull request merged**